advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Cohesity DataProtect for PostgreSQL'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | Cohesity       |
+| **Partner Product**   |  Cohesity DataProtect for PostgreSQL      |
+| **Web Site**      | https://www.cohesity.com/     |
+| **Version**   | Cohesity 6.6.0d       |
+| **Product Description**      | Cohesity simplifies database backup and recovery by eliminating multiple point products with a single software for on-premises or as-a-Service deployment. Protect and manage EDB Postgres Advanced Server and PostgreSQL data sources from a single global UI. |
```

Comment:
Need a new Product Description.  That one is not working.  Mostly the second sentence.  I am sure we are not protecting and managing the database.

Reply:
"Simplify protection of EDB Postgres Advanced Server and PostgreSQL data sources from a single global UI." Do we want to get into a lot of detail or keep this more simple?

Reply:
"Cohesity simplifies database backup and recovery by eliminating multiple point products with a single software for on-premises or as-a-Service deployment. Streamline protection of EDB Postgres Advanced Server and PostgreSQL data sources from a single global UI."

Current, updated product description.  @scottatchue